### ISSUE TITLE
Plugin should allow configuring multiple InfluxDB datasources 

### ISSUE THREAD

**Issue by pmouawad** [jenkinsci/influxdb-query-plugin#1]
First thanks for your plugin.

IMU, the plugin currently only allows hitting one InfluxDB since connection is configured once.

It would be useful to allow multiple connections:

- Either in Global Configuration by naming the configured connection and then passing it in query
- Or by allowing configuration at job query level but that might repeat connection configuration multiple times

Regards
Philippe M.

**Comment by JoeOffenberg** [comment]
Yes.  This is a good suggestion.  I’ll see about implementing it.

**Comment by pmouawad** [comment]
Hello,
Maybe my PR was misleading, but it doesn't fix #1, but #3 .
So could you reopen this one please ?

Thanks
